README.md

```diff
@@ -49,7 +49,7 @@ built by `fastly compute build`. The Fastly CLI should put the blob at
 viceroy bin/main.wasm
 ```
 
-This will start a local server (by default at: `http://127.0.0.1:7878`), which can
+This will start a local server (by default at: `http://127.0.0.1:7676`), which can
```

Comment:
Just to confirm that yes the CLI uses port 7676 by default:
https://github.com/fastly/cli/blob/main/pkg/commands/compute/serve.go#L78